_posts/2023-11-02-release.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Release 2023.44
+author: Priya
+date: 2023-11-02
+category: Jekyll
+layout: post
+---
+### Enhancements:
+
+#### Date Difference Calculation - Now with System Time and Specific Date Options:
+Calculate date differences more effectively with additional choices. You can choose between system time, specific dates, and even set calculations to the current date-time in UTC. The date difference component is also conveniently displayed in pipelines (Years, Months, Weeks, etc).
```

Comment:
The date difference rule now allows to calculate date difference between values in a date column and current time.